smurfs/src/components/SmurfForm.js

```diff
@@ -0,0 +1,45 @@
+import React        from "react";
+import { connect }  from "react-redux";
+import { addSmurf } from "../actions"
+
+ class SmurfForm extends React.Component {
+    constructor(){
+        super();
+        this.state = {
```

Comment:
these strings with a space in them is the reason your input placeholders do not generate until you have added something first and the setState triggers that sets them to empty strings "" instead of a string with a space " "